src/components/ITSBStore/itsbGraph.js

```diff
@@ -110,8 +130,30 @@ export class ITSBGraph {
     this.graph.endUpdate();
   };
 
-  getNode = (id) => this.graph.getNode(id)?.data;
+  /**
+   * Checks if one or more nodes with the given ID(s)
+   * exist in the graph. The method accepts multiple arguments.
+   * You can check for a single ID by calling `exists('id1')`,
+   * or for multiple IDs by calling `exists('id1', 'id2', 'id3').
+   * The return value will be `true` if ALL IDs exist in the graph.
+   *
+   * @param {...string} arg one or multiple node ids
+   * @returns {boolean} true if ALL of the given IDs exist
+   */
+  exists() {
```

Comment:
This is a JS pattern that I rarely see used! Do you prefer this over a ["rest" parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters)? Generally I prefer the latter.

Reply:
Ah - well, to my shame, I must admit I wasn't aware the rest parameter exists :-) I agree - that's really a nicer solution. I'm changing it.